AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image.js

```diff
@@ -39,6 +45,13 @@ define([
                 return record.styles;
             }
             return {};
+        },
+
+        /**
+         * Expand image preview
+         */
+        expandPreview: function (record) {
+            this.previewComponent().show(record._rowIndex);
```

Comment:
While the preview is shown, the thumbnail should be hidden

Reply:
We haven't requirements to hide the thumbnail image during the preview but we can improve this behavior in the scope of the separate task.